src/component/archive/ArchiveTable/ArchiveTable.tsx

```diff
@@ -153,12 +158,16 @@ export const ArchiveTable = ({
                 Header: 'Actions',
                 id: 'Actions',
                 align: 'center',
-                maxWidth: 85,
+                maxWidth: 100,
```

Comment:
Noticed the delete button overflowed the cell a bit:
![image](https://user-images.githubusercontent.com/14320932/180948266-71b8049d-c7c7-4679-ad47-db580b5ff7e9.png)

```suggestion
                maxWidth: 120,
```

Reply:
Fixed